src/R/Components/Impl/InteractiveWorkflow/Implementation/RInteractiveEvaluator.cs

```diff
@@ -74,18 +75,20 @@ public sealed class RInteractiveEvaluator : IInteractiveEvaluator {
 
         public async Task<ExecutionResult> ResetAsync(bool initialize = true) {
             try {
-                CurrentWindow.TextView.VisualElement.SizeChanged -= VisualElement_SizeChanged;
+                if (CurrentWindow != null) {
```

Comment:
If `CurrentWindow == null`, this is an automatic failure. Since there is a `catch (Exception ex)` around this block, we can skip these checks